Components/Page Elements/MoveForm.razor

```diff
@@ -0,0 +1,159 @@
+﻿@inject IMoveService moveService
+@inject IStringLocalizer<SharedResource> res
+@inject ISnackbar snackbar
+@inject IDialogService dialogService
+@inject IImageService imageService
+@inject NavigationManager nav
+
+@if (isLoading)
+{
+    <Loader />
+}
+else
+{
+    <EditForm Model="move" OnValidSubmit="HandleValidSubmit">
+        <DataAnnotationsValidator />
+        <ValidationSummary />
+        <MudGrid>
+            <MudItem xs="10">
+                <MudTextField @bind-Value="move.Name" Label="@res["Name"]" />
+            </MudItem>
+            <MudItem xs="2" Class="justify-end">
+                <MudText>@res["DifficultyLevel"]</MudText>
+                <MudRating SelectedValue="0" SelectedValueChanged="SetDifficulty" @bind-Value="move.DifficultyLevel" />
+            </MudItem>
+            <MudItem xs="12">
+                <MudTextField @bind-Value="move.Description" Lines="5" Label="Opis figury:" Variant="Variant.Outlined" />
+            </MudItem>
+            <MudItem xs="12">
+                <MudGrid>
+                    @if (urlImages.Any())
+                    {
+                        <MudItem xs="12">
+                            <MudText>@res["IncludedMedia"]:</MudText>
+                        </MudItem>
+                        @foreach (var imageUrl in urlImages)
+                        {
+                            <MudItem xs=@(urlImages.Count > 3 ? 4 : (urlImages.Count == 2 ? 6 : 12))>
+                                <img style="
+    width: 100%;
```

Comment:
Domyślam się, że moze jakieś problemy z dziedziczeniem CSS i przez to tutaj dałeś, jak nie da rady to zostaw, ale jak będzie opcja to możesz przenieść